src/traces/histogram/attributes.js

```diff
@@ -71,14 +71,44 @@ module.exports = {
         ].join(' ')
     },
 
-    mode: {
+    cumulative: {
+        valType: 'boolean',
+        dflt: false,
+        role: 'info',
+        description: [
+            'If true, display the cumulative distribution by summing the',
+            'binned values. Use the `direction` and `centralbin` attributes',
+            'to tune the accumulation method.'
+        ].join(' ')
+    },
+
+    direction: {
         valType: 'enumerated',
-        values: ['density', 'cumulative'],
-        dflt: 'density',
+        values: ['increasing', 'decreasing'],
+        dflt: 'increasing',
         role: 'info',
         description: [
-            ''
-        ].join('')
+            'Only applies if `cumulative=true.',
+            'If *increasing* (default) we sum all prior bins, so the result',
+            'increases from left to right. If *decreasing* we sum later bins',
+            'so the fresult decreases from left to right.'
+        ].join(' ')
+    },
```

Comment:
`direction='decreasing'` is to invert the accumulation - ie if you want "how much of the distribution is past this point" instead of "how much is before this point" (although note that it's not *exactly* the sum minus the increasing CDF unless you choose `currentbin='exclude'` for one, or `currentbin='half'` for both (see below).

Thoughts on the name `direction`? I'm not super excited about it but it seems OK.